docs/CONTRIBUTING.md

```
@@ -15,6 +15,12 @@ To set up a development environment, please follow these steps:
    cd starknet.quest
    npm i && npm run dev
    ```
+### 
+To avoid any failure to resolve the dependency tree, the --force flag should be added:
+
+```
```

Markdownlint

Fenced code blocks should have a language specified

(MD040, fenced-code-language)